lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -608,27 +875,57 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
         {min_full_name, _min_abbreviated_name} = min_station_name
         {max_full_name, _max_abbreviated_name} = max_station_name
 
-        if min_full_name == max_full_name do
-          "at #{min_full_name}"
-        else
-          "between #{min_full_name} and #{max_full_name}"
-        end
+        [min_full_name, max_full_name]
+    end
+  end
+
+  def format_endpoint_string(nil), do: nil
+
+  def format_endpoint_string([min_station, max_station]) do
```

Comment:
Is there a reason why the this function's non-nil arg is a list instead of a tuple? Usually when the number of elements is fixed, a tuple is the type to use. (As in `format_endpoint_string_as_freetext/1` below, for example)

Reply:
This was changed mostly for serialization. All existing functions that use this work with a tuple or a list, but in fullscreen alerts we are adding the result of this function to the response. I know Jason doesn't encode tuples without some extra work so I felt this was the easiest solution. Open to pushback on it though.

Reply:
Ok, got it. Would there be any way to keep it as a tuple for longer and convert to a list (`Tuple.to_list/1`) only when dropping it into the serialized map? If not/if that would complicate things too much, feel free to leave it as-is.

Reply:
Do you think it's worth finally implementing a custom encoder for tuples? This is something we run into every once in a while and from what I remember, we are usually good with tuples being serialized into lists. We could just tackle that quickly so that we can leave the result as a tuple while also allowing the responses to contain the result with no direct conversions.

Reply:
I think tackling this now is a good call. I _think_ the solution I came up with is fine. Just take a GL route and say it is `Green` always.